src/test/scala/com/exini/dicom/data/CompressionTest.scala

```diff
@@ -0,0 +1,26 @@
+package com.exini.dicom.data
+
+import akka.util.ByteString
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class CompressionTest extends AnyFlatSpec with Matchers {
+
+  "Compressing a dataset" should "reduce its size" in {
+    val bytes   = ByteString(Array.fill[Byte](256)(1)) ++ ByteString(Array.fill[Byte](256)(2))
+    val zipped1 = Compression.compress(bytes)
+    val zipped2 = Compression.compress(bytes, gzip = true)
+    zipped1.length should be < bytes.length
+    zipped2.length should be < bytes.length
+  }
+
+  it should "be perfectly restored when decompressed again" in {
+    val bytes   = ByteString(Array.fill[Byte](256)(1)) ++ ByteString(Array.fill[Byte](256)(2))
```

Comment:
Could fill with random bytes here to ensure it doesn't just work for this simple case.

Reply:
Fixed